cloudlift/deployment/service_template_generator.py

```diff
@@ -492,6 +661,52 @@ def _add_service_listener(self, service_name, target_group_action,
             self.template.add_resource(http_redirection_listener)
         return service_listener
 
+    def _add_nlb_alarms(self, service_name, nlb):
```

Comment:
this feels like code duplication, it doesn't change how we monitor nlb vs alb?

Reply:
Good point too.  Will update shortly.

Reply:
Refactored the 2 common LB alarms into a method `_add_common_lb_alarms` called by `_add_*_alarms`.